runtime/common/marshaling/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum MarshalingInjectedKeys {
+  IsError = '__is_marshaled_error__',
+  ErrorClassName = '__error_class_name__',
+  ErrorClassType = '__error_class_type__',
+
+  IsBuffer = '__is_marshaled_buffer__',
```

Comment:
Nit, any reason to do separate keys for each of these rather than have a single key that's something like `__marshaled_value_type__` with values like `'number', 'buffer', 'date'`, etc? Seems like it scales a bit better to new marshalable types.

Reply:
good idea.